composer/lib/dependabot/composer/helpers.rb

```diff
@@ -43,8 +43,6 @@ module Helpers
           .returns(String)
       end
       def self.composer_version(composer_json, parsed_lockfile = nil)
-        v1_unsupported = Dependabot::Experiments.enabled?(:composer_v1_unsupported_error)
-
         # If the parsed lockfile has a plugin API version, we return either V1 or V2
         # based on the major version of the lockfile.
         if parsed_lockfile && parsed_lockfile["plugin-api-version"]
```

Comment:
@sachin-sandhu , 

Is it possible to check like something following. We need to be sure if `plugin-api-version` version defined properly and it is lower or equal then `1` then it should be `V1` which will help us to throw unsupported error, otherwise V2?


```
if parsed_lockfile && parsed_lockfile["plugin-api-version"]
  version = Composer::Version.new(parsed_lockfile["plugin-api-version"])
  major_version = version.canonical_segments.first

  return major_version.nil? || major_version > 1 ? V2 : V1
end
```

Reply:
@kbukum1, i have fixed this